packages/shared-internals/src/ember-standard-modules.ts

```diff
@@ -37,6 +37,11 @@ emberVirtualPackages.add('@embroider/macros');
 // the modules-api-polyfill. Newer APIs need to be added here.
 emberVirtualPackages.add('@ember/owner');
 
+// these are not public API but they're included in ember-source, so for
+// correctness we still want to understand that they come from there.
+emberVirtualPackages.add('@glimmer/validator');
```

Comment:
should all the glimmer packages from ember-source be listed here?
- `@glimmer/compiler`
- `@glimmer/syntax`
- etc?

Reply:
It would be good to be complete. Also it would be good to make these lists ember-version-dependent.